apps/modernization-ui/src/apps/page-builder/pages/BusinessRulesLibrary/BusinessRulesForm.tsx

```diff
@@ -176,7 +183,13 @@ const BusinessRulesForm = () => {
                 render={({ field: { onBlur, onChange, value }, fieldState: { error } }) => (
                     <Grid row className="inline-field">
                         <Grid col={3}>
-                            <label className="input-label">Logic</label>
+                            <label className="input-label">
+                                Logic{' '}
+                                <RequiredFieldMarker
+                                    functionName={form.watch('ruleFunction') || 'default'}
+                                    fieldName="comparitor"
+                                />
+                            </label>
```

Comment:
Issue: The `SelectInput` already provides the required indicator. All you need to do is give it the `label` as well as the `required`.

Ex:
```
 <SelectInput
    label="Subgroup"
    defaultValue={value}
    onChange={(e) => {
        onChange(e);
        onBlur();
    }}
    onBlur={onBlur}
    error={error?.message}
    options={subgroups}
    name={name}
    htmlFor={name}
    id={name}
    disabled={editing}
    required
/>
```
![image](https://github.com/CDCgov/NEDSS-Modernization/assets/109251240/416f0c71-1f40-4d94-a84f-39ce865065fb)